content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -14,5 +14,34 @@ match:
 tooltip: >-
   Initialized by running `dvc init` in the **workspace** (typically a Git
   repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  `.dvc` files created with commands such as `dvc add` or `dvc run`.  
+  See [full concept](/doc/user-guide/basic-concepts/dvc-project).
 ---
+
+# DVC Project
+
+Initialized by running `dvc init` in you <abbr>workspace</abbr> directory, it
+will contain all the [DVC files and directories][dvc-metafiles] including the
+<abbr>cache</abbr>, `dvc.yaml` and `.dvc` files, etc. Any other files referenced
+from these DVC metafiles are also considered part of the project (for example
+[metrics files](/doc/command-reference/metrics)).
```

Comment:
Thanks for the suggestion but I think both are valid since this text is right under the **DVC Project** title (so that's clearly what "it" refers to, I think). Stylistic

Reply:
Oh I see there's another copy edit that changes `"including" for long dash. Same thing though, both are valid I think. I incline to keep mine of course unless there's an error somewhere.

Reply:
```suggestion
Initialized by running `dvc init` in your <abbr>workspace</abbr> directory, a
DVC project will contain all of the [DVC metafiles and
directories][dvc-metafiles], such as the <abbr>cache</abbr>, `dvc.yaml`, and
`.dvc` files. Any other files referenced from these DVC metafiles are also
considered part of the project (e.g. [metrics
files](/doc/command-reference/metrics)).
```

lots of typos as well. compromised.

Reply:
What typos?